python/lsst/obs/base/yamlCamera.py

```diff
@@ -105,32 +105,25 @@ def makeDetectorConfigList(ccdParams):
     return detectorConfigs
 
 
-def makeAmpInfoCatalog(ccd):
-    """Construct an amplifier info catalog
+def makeAmplifierList(ccd):
```

Comment:
Is this routine still super slow? If you look at `test_ingest.py` in `obs_lsst` you will see it can take a very long time (run `pytest --durations=20`).

Reply:
The results on both ticket branch and master result in essentially the same timings, so this seems to be equally slow.